lib/vet360/contact_information/service.rb

```diff
@@ -107,8 +107,8 @@ def post_or_put_data(method, model, path, response_class)
 
         with_monitoring do
           raw_response = perform(method, path, model.in_json)
-          Vet360Redis::Cache.invalidate(@user)
 
+          Vet360Redis::Cache.invalidate(@user)
```

Comment:
In keeping with the same logic of cache invalidation happening at the controller layer, should this also live in the controller(s)? I definitely see the convenience of doing it here. Is there precedence for this in other service clients?